### DISCUSSION TITLE
Mac m1 ?

### DISCUSSION THREAD

**Discussion by trinhpxl** [keiwando/toonsquid#57] [Q&A]
ooo, apparently my mom is thinking of investing ina new mac sometime this year or next ! and i would wanna use her mac sometimes if toonsquid becomes available on the mac ! :3 i hope more people request the mac m1 version of toonsquid :D also, just a general q bc im not an expert on this and im curious to have insight and sorry if my question is too bold or intrusive - 

but is there anyway you'd be working with a team in the future ? or are you planning to trooper it out and solo this mission (making toonsquid by yourself) ? 

(how long does it take to know coding for ios BC IM BOUTTA SPEEDRUN LEARNING TO HELP LMAO /JOKING IM JOKING)

**Comment by LoneStarz** [comment]
I think ipad apps are already available on Mac m1 I’m not sure tho! I’m also thinking of getting an m2 MacBook air

**Reply by keiwando** [reply]
The developers get to choose whether they want their iPad apps to be available for M1 Macs as well or not. There are definitely still quite a number of things I would need to do before I'd be happy with the macOS version of ToonSquid (since you'll naturally want to use it differently than on an iPad).

**Reply by jordanwmartin** [reply]
@keiwando This sounds like you were working on an M1 version. Is this something we might see eventually?

**Reply by twinkypix** [reply]
Now that Adobe has essentially abandoned Animate, I think this is a prime opportunity to grab those Mac animators who are looking to switch from Animate. I am also a developer for iOS, and yes we are able to make a Mac version available when publishing. However, realize that a Mac version will require modifications to work properly without touch. I would GLADLY purchase a separate Mac version, and I don't think I'm alone on this.

**Comment by wpd2012** [comment]
The recent update is really nice, It's ready for macOS I believe, sure there are some adjustments I would prefer for the desktop version, but so far this is already sellable

**Comment by jordanwmartin** [comment]
There are some things I would want before this, but being able to work seamlessly between iPad and Mac would be unbelievably incredible.

**Comment by diegostoliar** [comment]
I think its a nobrainer to have toonsquid on mac m1... even if I need to pay again. All the other options to animate on MacOs are obsenely expensive... Only Krita is a real competitor, but I think ToonSquid actually already have some advantages compared to Krita... so I would totally buy it again on MacOs.

**Reply by amoraleite** [reply]
+1
I know how's difficult is to work on two or more platforms, but for a Mac version it’s too close! 
BTW @keiwando thank you for this tool it’s really a exceptional tool.
